release/pkg/helm/helm.go

```diff
@@ -74,6 +74,8 @@ func GetHelmDest(d *helmDriver, r *releasetypes.ReleaseConfig, ReleaseImageURI,
 	}
 
 	helmChart := strings.Split(ReleaseImageURI, ":")
+	HelmLog.Info("Starting to modifying helm chart %s", helmChart[1])
```

Comment:
I feel this should be helmChart[0] with the URI
```suggestion
	HelmLog.Info("Starting to modifying helm chart %s", helmChart[0])
```